apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -680,6 +655,42 @@ export class PortfolioService {
     };
   }
 
+  public async getSummary(aImpersonationId: string): Promise<PortfolioSummary> {
+    const currency = this.request.user.Settings.currency;
+    const userId = await this.getUserId(aImpersonationId);
+
+    const performanceInformation = await this.getPerformance(userId);
+
+    const { balance } = await this.accountService.getCashDetails(
+      userId,
+      currency
+    );
+    const orders = await this.orderService.getOrders({ userId });
+    const fees = this.getFees(orders);
+    const firstOrderDate = orders[0]?.date;
+
+    const totalBuy = this.getTotalByType(orders, currency, TypeOfOrder.BUY);
+    const totalSell = this.getTotalByType(orders, currency, TypeOfOrder.SELL);
+
+    const committedFunds = new Big(totalBuy).sub(new Big(totalSell));
```

Comment:
You don't need the second `new Big(...)`

Reply:
Thank you for the hint! 🙏🏻